components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/constants/AccountConstants.java

```diff
@@ -51,7 +51,7 @@
     public static final String LOCKED = "LOCKED";
     public static final String UNLOCKED = "UNLOCKED";
     public static final String DISABLED = "DISABLED";
-
+    public static final String USER_ACCOUNT_STATE_WITH_USERNAME = "UserAccountStateWithUsername";
```

Comment:
Any reason to add this constant? Can't we use https://github.com/wso2/carbon-identity-framework/pull/3216/files#diff-5398caecd73528910fe590141c693a88b807d6f029c2f16e61866e62515b70d3R66?